hooks/post_gen_project.py

```diff
@@ -83,7 +84,11 @@ def post_gen():
     subprocess.call(["git", "checkout", "-b", git_new_branch])
     subprocess.call(["git", "add", "."])
     subprocess.call(["git", "commit", "-m", "Initial commit"])
-
-
+    
+def code_of_conduct_clean_up():
+    move_selected_doc_dir()
```

Comment:
```suggestion
```

you don't need this function call here.
you should call `code_of_conduct_clean_up()` after https://github.com/osl-incubator/cookiecutter-python/pull/62/files#diff-8601710441b0dde909812d3f5a1d83245fec1f1bf94b6e510d1b72cbd2fb86c1R58

sorry if my previous comment was not super clear.